Dockerfile

```diff
@@ -76,6 +76,8 @@ MAILER_DSN=null://null \
 ILIOS_LOCALE=en \
 ILIOS_SECRET=ThisTokenIsNotSoSecretChangeIt \
 ILIOS_REQUIRE_SECURE_CONNECTION=false \
+ILIOS_SEARCH_HOSTS=http://opensearch:9200 \
```

Comment:
Adding this here will force open search on all consumer of the containers, which we don't want. Same with tika on the line below. These should be provided at runtime into the container like we do in compose.

Reply:
removed!